Feat inventory

karakune opened the pull request.

karakune committed:
[WIP] started work on inventory

karakune committed:
[WIP] started work on inventory

karakune committed:
[WIP] followed tutorial for inventory, clean up needed

karakune committed:
cleaned up code, added UI item counter

karakune committed:
started work on a quest related to inventories

karakune committed:
Completed inventory and related quest

karakune committed:
fixed missing dictionary UI

DrunkReaperMatt reviewed (changes requested):
![image](https://user-images.githubusercontent.com/5804478/70282149-07faee80-178b-11ea-847a-55ef491cd794.png)

BP dans inventory montre ceci au init

**EDIT:** ne pas edit. Le probleme etait dans la compilation du projet

DrunkReaperMatt reviewed (changes requested):
coconut 5 can't get

celui sur le toit

DrunkReaperMatt reviewed (commented):
prendre les 16 cononuts avant de parler a jake crash le jeu

ErikaElfassy committed:
NPC interact crash fix

I believe the gc of ue was getting rid of these values then trying to get them since there was no uproperty tag and no inilialization of the dictionary pointer
